Sources/Networking/HTTPRequest.swift

```diff
@@ -16,6 +16,7 @@ struct HTTPRequest {
 
     let method: Method
     let path: Path
+    let timeoutSeconds = Configuration.networkTimeoutDefault
```

Comment:
Is this needed here? I'm confused why this wouldn't use `HTTPClient.networkTimeout`

Reply:
AYyyyyyy oops.

Reply:
✅